dist/backbone.storage.js

```diff
@@ -39,12 +39,15 @@
      * @method find
```

Comment:
Please remove the dist files from the commit, that will happen as part of publishing